Geometry/CMSCommonData/python/cmsExtendedGeometry2023XML_cfi.py

```diff
@@ -233,7 +234,7 @@
         'Geometry/MuonCommonData/data/v7/gem21.xml',
         'Geometry/MuonCommonData/data/v2/csc.xml',
         'Geometry/MuonCommonData/data/PhaseII/mfshield.xml',
-        'Geometry/MuonCommonData/data/v8/me0.xml',
+        'Geometry/MuonCommonData/data/PhaseII/me0.xml',
```

Comment:
@lgray - ok, no need to change the PR. I can see that @bsunanda's changes were reverted by @davidlange6: https://github.com/cms-sw/cmssw/commit/c5f6536ca5a7852f8a0425de9e1168f6c778999a